frontend/app/models/daterange-model.js

```diff
@@ -9,5 +9,10 @@ export default DS.Model.extend({
   isYearValid(year) {
     if (year.length == 4 && year > 0) return true;
     return false;
+  },
+
+  missesMonth() {
+    if (!this.monthFrom || (!this.monthTo && this.yearTo)) return true;
+    return false;
```

Comment:
`return !this.monthFrom || (!this.monthTo && this.yearTo)` should do the trick. Also maybe rename to missesMonths as in plural :)